spring-cloud-stream-module-launcher/src/main/java/org/springframework/cloud/stream/module/launcher/ModuleLauncherProperties.java

```diff
@@ -34,6 +47,21 @@
 	 */
 	private String[] modules;
 
+	private List<ModuleWithArguments> modulesWithArguments = new ArrayList<>();
+
+	private PropertySources propertySources;
```

Comment:
something doesn't seem right about having PropertySources themselves in ConfigurationProperties